### PR TITLE
DDT-983: Update PUT SI

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: src/test/java/org/dcsa/core/events/service/impl/ConsignmentItemServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,1063 @@
+//package org.dcsa.core.events.service.impl;
```
Comment (pimg) [inline src/test/java/org/dcsa/core/events/service/impl/ConsignmentItemServiceImplTest.java:1]: All these tests are commented out, why are they part of this PR in this form?

File: src/main/java/org/dcsa/core/events/edocumentation/service/impl/ConsignmentItemServiceImpl.java [hunk 2]
```diff
@@ -99,6 +97,39 @@
...
+                    .flatMap(
+                        consignmentItem ->
+                            referenceRepository
+                                .deleteByConsignmentItemID(consignmentItem.getId())
+                                .flatMap(
+                                    x ->
+                                        consignmentItemRepository.deleteById(
+                                            consignmentItem.getId()))))
+        .collectList()
+        .flatMap(ignored -> Mono.empty());
```
Comment (pimg) [inline src/main/java/org/dcsa/core/events/edocumentation/service/impl/ConsignmentItemServiceImpl.java:130]: I don't understand this part. Does it return a Mono.empty() or a list of consignment items?

File: src/test/java/org/dcsa/core/events/service/impl/ConsignmentItemServiceImplTest.java (hunk 1 above)
Comment (mp-assco) [reply src/test/java/org/dcsa/core/events/service/impl/ConsignmentItemServiceImplTest.java:1]: Mistake. It was supposed to be in a separate PR for consignment tests.

File: src/main/java/org/dcsa/core/events/edocumentation/service/impl/ConsignmentItemServiceImpl.java (hunk 2 above)
Comment (mp-assco) [reply src/main/java/org/dcsa/core/events/edocumentation/service/impl/ConsignmentItemServiceImpl.java]: Empty. Made clearer now.

File: src/test/java/org/dcsa/core/events/service/impl/ConsignmentItemServiceImplTest.java (hunk 1 above)
Comment (mp-assco) [reply src/test/java/org/dcsa/core/events/service/impl/ConsignmentItemServiceImplTest.java]: Removed

**Review by pimg — APPROVED** [review]

**Pull request merged**